### PR TITLE
[#EP-2111] Add MFA support to sign-in.

### PR DESCRIPTION
**Pull request by Omicron7**
This adds Multi-factor support to Sign-In.
It should continue to work without the cortex-gateway changes, it just won't show the MFA token field.

### CODE REVIEW

**Comment by Omicron7** [comment]
Cortex Gateway change: https://github.com/CruGlobal/cortex_gateway/pull/52

File: src/common/components/signInForm/signInForm.tpl.html [hunk 1]
```diff
@@ -2,34 +2,56 @@
...
-           placeholder="{{'Password'|translate}}"
-           autocomplete="off"
-           required
-           ng-disabled="$ctrl.isSigningIn"
-           ng-model="$ctrl.password">
+  <div ng-switch-when="mfa">
+    <p translate>Enter a verification code from your Authenticator app to continue.</p>
+    <div class="form-group">
+      <label for="mfa_token" translate>Verification Code</label>
+      <input id="mfa_token"
```
Comment (OzzieOrca) [inline src/common/components/signInForm/signInForm.tpl.html]: @Omicron7 Can we add autofocus here? I just used it for the first time and thought that would be nice.

Comment (adammeyer) [reply src/common/components/signInForm/signInForm.tpl.html]: 👍

Comment (Omicron7) [reply src/common/components/signInForm/signInForm.tpl.html]: @OzzieOrca You can indeed add autofocus there.

**Pull request merged**